src/server/acl/acl_family.cc

```diff
@@ -48,12 +50,16 @@ void AclFamily::Acl(CmdArgList args, ConnectionContext* cntx) {
 void AclFamily::List(CmdArgList args, ConnectionContext* cntx) {
   const auto registry_with_lock = ServerState::tlocal()->user_registry->GetRegistryWithLock();
   const auto& registry = registry_with_lock.registry;
-
   (*cntx)->StartArray(registry.size());
+
+  auto pretty_print_sha = [](std::string_view pass) {
+    return absl::BytesToHexString(pass.substr(15)).substr(15);
```

Comment:
I don't understand 😄 bytes to hex string returns a string of length 30 (twice the size), why do t
you need only half of it?

Reply:
Okay I'm just gonna trust you on this one 🤣